src/components/cell-tooltip/CellTooltipText.js

```diff
@@ -30,12 +30,14 @@ export default function CellTooltipText(props) {
       }}
     >
       <table>
-        {Object.keys(factors).map(key => (
-          <tr>
-            <th>{key}</th>
-            <td>{factors[key]}</td>
-          </tr>
-        ))}
+        <tbody>
+          {Object.keys(factors).map(key => (
```

Comment:
`Object.entries` is good for cases like this. (It's the inverse of the `fromentries` you just added.)